apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -160,6 +192,28 @@ const PollWorkerScreen = ({
             </Prose>
           }
         />
+        <Modal
+          isOpen={isConfirmingEnableLiveMode}
+          centerContent
+          content={
+            <Prose textCenter>
+              <p>
+                {isPrintMode
+                  ? 'Switch to Live Election Mode and reset the tally of printed ballots?'
+                  : 'Switch to Live Election Mode?'}{' '}
+                You will need an Admin card to switch back to Testing Mode.
+              </p>
+            </Prose>
+          }
+          actions={
+            <React.Fragment>
+              <Button primary onPress={confirmEnableLiveMode}>
```

Comment:
Override "primary" style with "danger" style when action may be destructive: 
```suggestion
              <Button primary danger={isPrintMode} onPress={confirmEnableLiveMode}>
```

Reply:
![127 0 0 1_3000_(Surface Go)](https://user-images.githubusercontent.com/1938/98308879-0549e000-1f7e-11eb-8eda-ffe46f0f3725.png)